src/calendar/calendar-day.style.tsx

```diff
@@ -0,0 +1,180 @@
+import { Color } from "src/color";
+import { TextStyleHelper } from "src/text";
+import styled, { css } from "styled-components";
+import { VariantDay } from "./types";
+
+interface DayNumerProps {
+    variant: VariantDay;
+    disabled: boolean;
+}
+
+interface CircleProps extends Pick<DayNumerProps, "variant"> {
+    position: "left" | "right";
+}
+
+interface InteractiveCircleProps extends Omit<DayNumerProps, "variant"> {}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const WeekDaysContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin-bottom: 0.5rem;
+`;
+
+export const WeekCell = styled.div`
+    ${TextStyleHelper.getFontFamily("H6", "semibold")}
```

Comment:
use `TextStyleHelper.getTextStyle` instead if you need the matching font-size

Reply:
Thanks for sharing this method. This is what i need.